api/delete-histogram.md

```diff
@@ -8,32 +8,43 @@ This call will return an empty array upon success. If there is an error, it will
 Description for API call
 ------------------------
 
-**URI:**   /histogram/&lt;end&gt;/&lt;uuid&gt;/&lt;metric&gt;
+**URI:**   /histogram/&lt;uuid&gt;/&lt;metric&gt;
 
 **Method:**   DELETE
 
 **Inputs:**
 
-*end* :   The end time. All data from before this specified time is deleted. Time is represented in seconds since the epoch.
-
 *uuid* :   The UUID of the check to which the metric belongs.
 
 *metric* :   The name of the metric from which to delete data.
 
+*end* :   The end timestamp for the delete operation. All data from before this specified time is deleted. Time is represented in seconds since the epoch.
```

Comment:
This is no longer an input so I'd remove it under inputs to avoid confusion.
You can copy the text under header section to preserve the explanation.